Felienne/hedy
# Small fixes on admin /stats page #1670

**Description**
Multiple small fixes on the admin /stats page
See comment in #1670 for why this PR is not migrating the date column from string to timestamp.

**Fix for**
#1670 

**How to test**

Unfortunately, logs can only be tested on alpha.
- Open the /stats page and check that the start and end date input fields of the log search are correctly pre-filled with today's date and time.
- Perform a query that returns no data. Now press the search button. Note that the message that the previous query did not yield any results will disappear.
- When the username of a log record is not present, the cells should contain nothing instead of the value 'null'.
- Query search results should be sorted by date.
